gig/migrations/0013_gig_email_changes_historicalgig_email_changes_and_more.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 4.2.10 on 2024-03-09 17:13
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('band', '0014_alter_band_default_language'),
+        ('gig', '0012_auto_20210604_1736'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='gig',
+            name='email_changes',
+            field=models.BooleanField(default=True),
+        ),
+        migrations.AddField(
+            model_name='historicalgig',
+            name='email_changes',
+            field=models.BooleanField(default=True),
```

Comment:
```suggestion
```suggestion
        migrations.RenameField(
            model_name='historicalgig',
            old_name='send_update',
            new_name='email_changes',
```
```